### PR TITLE
chore: update auto-updater to configurable version

### PR DESCRIPTION
**Pull request by castastrophe**
## Description

The auto-updater we were using was not configurable to allow us to control what PRs get updated so I'm swapping it for this one that I forked that adds the functionality we need. 

- Filters out draft PRs
- Limits to 50 PRs to update per push
- Excludes PRs with blocked or wip labels

Moved the skip_vrt label to the Chromatic skip input which will pass PRs when the label is present. This will let us put the VRT as a required action.

## To-do list
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).
- [x] This pull request is ready to merge.

### CODE REVIEW

File: .github/workflows/pr-update.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: maxkomarychev/pr-updater-action@v1.0.1
+      - uses: castastrophe/actions-pr-auto-update@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          include_drafts: false
```
Comment (pfulton) [inline .github/workflows/pr-update.yml:17]: Configurable feels much better!

We should document this option somewhere (and also communicate it to the team)

File: .github/workflows/pr-update.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ jobs:
...
+          limit: 50
+          exclude_labels: blocked,wip
```
Comment (pfulton) [inline .github/workflows/pr-update.yml:19]: We should document this option somewhere, too (and also communicate it to the team)

**Review by pfulton — APPROVED** [review]

File: .github/workflows/pr-update.yml (hunk 1 above)
Comment (castastrophe) [reply .github/workflows/pr-update.yml:17]: Great point. The documentation is written up here but I'll add a comment to that effect in here.

File: .github/workflows/pr-update.yml [hunk 3]
```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: maxkomarychev/pr-updater-action@v1.0.1
+      - uses: castastrophe/actions-pr-auto-update@v1
```
Comment (castastrophe) [inline .github/workflows/pr-update.yml:14]:
```suggestion
       # README: https://github.com/castastrophe/actions-pr-auto-update#auto-update-pull-requests
      - uses: castastrophe/actions-pr-auto-update@v1
```

File: .github/workflows/pr-update.yml (hunk 1 above)
Comment (castastrophe) [inline .github/workflows/pr-update.yml:17]:
```suggestion
          # Draft PRs will not be automatically updated by this utility
          include_drafts: false
```

**Pull request merged**